docs/js/main.js

```diff
@@ -13,6 +14,19 @@ import Root from './Root';
 import '../css/demo.scss';
 import '../img/favicon.ico';
 
+(() => injectGlobal`
+  ${styledNormalize}
+
+  /* https://github.com/necolas/normalize.css/issues/694 */
+  button,
+  input,
+  optgroup,
+  select,
+  textarea {
```

Comment:
Where did you take the list of elements from? why is it that e.g. `a` and `span` don't have to be listed?

Reply:
from the css linked from the issue